wdlconfig.qmd

```diff
@@ -0,0 +1,58 @@
+
+# WDL Configuration Guide {{< iconify file-icons wdl >}} {#sec-wdlconfig}
+
+So as not to reinvent the wheel, WILDS WDL's should follow guidelines similar to the ones provided by [BioWDL](https://biowdl.github.io/styleGuidelines.html) and [WARP](https://broadinstitute.github.io/warp/docs/Best_practices/suggested_formats). However, because of the pedagogical "proof-of-concept" nature of the WILDS, they will not be identical and even differ significantly in a few places.
+
+## WILDS Philosophy
+
+- The mindset behind WILDS is for each repository to be a self-contained demonstration of a particular bioinformatic functionality:
```

Comment:
Is this the philosophy of WILD or WDL WILDS?
Alternative sentence idea:
Each WILDS repository is ideally a self-contained demonstration of a specific bioinformatic workflow.

Reply:
Good point, probably best to be more specific in saying "WILDS WDLs" rather than WILDS in general.